tools/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 list( APPEND programs
     ${PROJECT_NAME}_lint.sh
     ${PROJECT_NAME}_comp.sh
+    ${PROJECT_NAME}_cpplint.py
```

Comment:
Can we possibly add this to jedicmake.  At the moment, every repo is contributing its own version.  I can submit a PR to jedicmake to export this maybe?  I guess then it would require jedicmake to run this test?  Or we could just go ahead and make jedicmake a required dependency?

Reply:
I copied this from oops. I believe UFO, IODA have their own copies.
We can add this to jedicmake and make it a required repo.  However this is not cmake related and thus not really a jedicmake thing.

Reply:
I forget the reason why we split cpplint per repository. It was pre-jedicmake. The versions have diverged: https://github.com/JCSDA/oops/commits/develop/tools/cpplint.py, https://github.com/JCSDA/ufo/commits/develop/tools/cpplint.py.

There was an old ecbuild issue where the wrong cpplint.py was being run, which is why we settled on different names.

Reply:
We'll what's most important is that the cpplint.py doesn't have any dependencies so adding a module to jedicmake to provide it is easy, and will reduce duplication and confusion.  We want to bump common stuff up to that level.  Eventually all the "extra macros" will be unified and added there too to lessen the maintenance burden on each repo to maintain that infrastructure.

Reply:
> We'll what's most important is that the cpplint.py doesn't have any dependencies so adding a module to jedicmake to provide it is easy, and will reduce duplication and confusion. We want to bump common stuff up to that level. Eventually all the "extra macros" will be unified and added there too to lessen the maintenance burden on each repo to maintain that infrastructure.

Ok. So what do you want me to do here? 
Should I wait until this is added to `jedicmake`?